junwoo/app.js

```diff
@@ -29,7 +30,38 @@ app.get("/ping", cors(), (req, res) => {
   res.json({ message: "pong" });
 });
 
-const PORT = process.env.PORT;
+app.post("/users/signup", async (req, res) => {
+  const { name, email, age } = req.body;
+
+  const result = await appDataSource.query(
+    `INSERT INTO users(
+      name,
+      email,
+      age
+      ) VALUES (?,?,?);
+      `,
+    [name, email, age]
+  );
+  console.log(result);
+  res.status(201).json({ message: "userCreated" });
+});
+
+app.post("/posts/create", async (req, res) => {
+  const { title, content, image, user_id } = req.body;
+
+  await appDataSource.query(
+    `INSERT INTO posts(
+  title,
+  content,
+  image_url,
+  user_id
+  ) VALUES(?,?,?,?);
+  `,
+    [title, content, image, user_id]
+  );
```

Comment:
- 우선 raw query의 포매팅을 수정하면 좋을 것 같습니다! 동일한 프로젝트에서 동일한 포매팅을 유지함으로써 코드 구조를 파악하고 코드를 읽기도 용이해질 것 같습니다!

```suggestion
  await appDataSource.query(
    `INSERT INTO posts(
        title,
        content,
        image_url,
        user_id
      ) VALUES (
          ?,
          ?,
          ?,
          ?);`
        , [title, content, image, user_id]
  );
```